spec/spec.md

```diff
@@ -50,6 +50,12 @@ WACI v0.1 is a _PRE-DRAFT_ specification under development by the [DIF Claims an
 
 TBD
 
+## Selective Disclosure
+
+To allow for selective disclosure of Verifiable Credential claims the use of a JSON-LD frame object is combined with a `BbsBlsSignature2020` on the credential. This signature allows for a zero knowledge proof of the original signature, allowing the prover to derive a `BbsBlsSignatureProof2020` that allows verification of the disclosed claims.
+
+More information about how this works can be found in the [Linked Data Proof BBS+ Signatures 2020 Suite](https://w3c-ccg.github.io/ldp-bbs2020/#the-bbs-signature-proof-suite-2020) and the [Mattr example implementation](https://github.com/mattrglobal/jsonld-signatures-bbs).
```

Comment:
I'd add another sentence to the "more info" paragraph, since some people prefer a little explanation along with their examples:
For a more general overview of LD-Framing strategies as a general approach to querying and matching Linked-Data, see the [JSON-LD Framing](https://json-ld.org/spec/FCGS/json-ld-framing/20180607/#framing) guide written by the JSON-LD Community Group at W3C on the occasion of version 1.1 of the JSON-LD specification.